src/Components/Leg/Leg.js

```diff
@@ -35,11 +35,20 @@ export const Leg = (props) => {
             <Text style={styles.dateText}>{endDate}</Text>
             </View>
           </View>
-          <View>
-          </View>
+          
+            <View style={styles.sideBySideContainer}>
+          <TouchableOpacity style={styles.sideBySideButton}>
+            <Text style={styles.buttonText} onPress={() => navigate('Transportation', { leg })}>Transportation</Text>
```

Comment:
We need to pass the userID in here so that the footer navigation in AddTransportInfo has the userId.

Test this by going to AddTransportInfo and then clicking on the Followers button in the footer --> It'll throw an error when fetching followers.